.web-docs/components/builder/vmx/README.md

```diff
@@ -1132,7 +1140,65 @@ Since ovftool is only capable of password based authentication
 - `vnc_disable_password` - This must be set to "true" when using VNC with
 ESXi 6.5 or 6.7.
 
-### SSH key pair automation
+### Building with the VMWare Workstation Pro API (vmrest)
+
+VMWare Workstation Pro ships with a 'vmrest' executable enabling programmatic 
```

Comment:
"VMware" for all occurrences of "VMWare", please.

Reply:
Thank you for the review! I've corrected "VMWare" to "VMware" in all instances where I wrote the former. 

(Note: I did not make the change for instances of "VMWare" that preexisted my contribution)